__tests__/src/index.spec.js

```diff
@@ -223,4 +223,44 @@ describe('toMatchImageSnapshot', () => {
     const matcherAtTest = toMatchImageSnapshot.bind(mockTestContext);
     expect(() => matcherAtTest('pretendthisisanimagebuffer')).not.toThrow();
   });
+
+  it('can provide custom defaults', () => {
```

Comment:
Can you change this test up so that the assertions are not within the mocks? And to get rid of all references to `blinkDiff`?